src/components/Modal/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Modal = ({
+  id, children, title, footer,
+}) => (
+  <div id={id} className="modal">
+    <div className="modal-content">
+      <h4>{ title }</h4>
+      { children }
+    </div>
+    {/* {
```

Comment:
Hey @bevkololi ,
I see you have commented out the `footer`.  Does this mean you won't be using it or you had a challenge that made you comment it out? In case of the problem, kindly share what was going wrong so we can try and assist.